x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/sources_logic.ts

```diff
@@ -50,7 +50,7 @@ export interface IPermissionsModalProps {
   additionalConfiguration: boolean;
 }
 
-type CombinedDataItem = SourceDataItem & ContentSourceDetails;
+type CombinedDataItem = SourceDataItem & SourceDataItem & { connected: boolean };
```

Comment:
The second `SourceDataItem` here seems off. Do we need it twice here?

Reply:
We definitely do not, thanks!